docs/en/latest/plugins/prometheus.md

```diff
@@ -0,0 +1,124 @@
+---
+title: prometheus
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This guide shows how to monitor Apache APISIX Ingress Controller using Prometheus and Grafana.
+
+## Enable Prometheus
+
+You can find example CRD files in `apisix-ingress-controller/samples/deploy/crd/v1`.  
+The CRD file `ApisixClusterConfig` is required to use to enable prometheus in global configurations.  
+In `ApisixClusterConfig`, `monitoring.prometheus` field is used to describe Prometheus settings.
+```yaml
```

Comment:
This file is a definition file for custom resources. In order to be able to configure correctly, we should create a custom resource configuration file, like this:

```yaml
apiVersion: apisix.apache.org/v2beta3
kind: ApisixClusterConfig
metadata:
  name: default
spec:
  monitoring:
    prometheus:
      enable: true
```

Reply:
please see if I understand the usage of crd file.

Reply:
looks good.